docs/installation/viam-server-setup.md

```diff
@@ -0,0 +1,125 @@
+---
+title: "viam-server Installation Guide"
+linkTitle: "viam-server setup"
+childTitleEndOverwrite: "Try Viam"
+weight: 9
+no_list: true
+type: docs
+images: ["/installation/thumbnails/install.png"]
+imageAlt: "Install Viam"
+description: "To use Viam, install and run viam-server or viam-micro-server on the computer or microcontroller that will run your machine and is connected to your hardware."
+date: "2024-08-19"
+# updated: ""  # When the content was last entirely checked
+aliases:
+  - /installation/prepare/
+  - /installation/macos-install/
+  - /installation/linux-install/
+  - /installation/install/
+  - /installation/install/linux-install/
+  - /installation/install/macos-install
+  - /getting-started/installation/
+  - /getting-started/macos-install/
+  - /getting-started/linux-install/
+  - /installation/
+  - /get-started/installation/
+---
+
+To use Viam, you need to install either the `viam-server` binary or `viam-micro-server`.
+
+[`viam-server`](/architecture/rdk/) is the binary built from the [Robot Development Kit](https://github.com/viamrobotics/rdk) that contains and manages communications between all Viam's built-in hardware drivers ({{< glossary_tooltip term_id="component" text="components" >}}) and software {{< glossary_tooltip term_id="service" text="services" >}}, connects your machine to the cloud, manages machine configuration, and manages dependencies including {{< glossary_tooltip term_id="module" text="modules" >}}.
+
+`viam-micro-server` is a lightweight version of `viam-server` which can run on resource-limited embedded systems that cannot run the fully-featured `viam-server`.
+To install `viam-micro-server`, see [`viam-micro-server` Installation Guide](/installation/viam-micro-server-setup/).
+
+For an overview of the Viam software platform, see [Learn about Viam](/get-started/).
```

Comment:
Really excited that these changes are going in! I just wanted to make one suggestion - I don't think it makes sense to mention viam-micro-server on this page which is otherwise about viam-server.

We could potentially keep the top-level "Installation Guide" page and just have it contain this text. (We could add a sentence at the end of the second pragraph "viam-server is the binary..." that links to this page, similar to the sentence at the end of the third paragraph "vim-micro-server is a lightweight...")

Reply:
Or we could just remove the part about viam-micro-server from here, I just saw it's repeated on the two viam-micro-server pages

Reply:
We can limit the explanation and really only mention the other pages. However, they do need to both mention each other. Because it's likely to be confusing for anyone unfamiliar with viam-server and viam-micro-server and they may end up on either page and then think we only support microcontrollers or SBCs. That's a reason to not separate the pages by the way. We may be hurting ourselves and I honestly still haven't decided whether this is the right thing to do here.

I think you were one of the people to point out that maybe people don't even click on the top level page. So I don't think it would be a good idea to put vital information for understanding on that page.

Reply:
after re-reading it just now, I do actually think this works as is

Reply:
What do you both think about changing the titles to `Installation (Microcontrollers)` and `Installation`?

Reply:
Sure, that works - I think with that name change it makes sense to have a link to the microcontrollers page from the main installation page

Reply:
Made this title change!